BolajiOlajide/a_socials
# Ft view event details on slack 165717871

#### What Does This PR Do?
- adds a button for view event details on slack

#### Description Of Task To Be Completed
As a user subscribed to a category, when I get Slack notifications about events happening in the category, I should be able to see a "View details" button such that when I click on the button, I am redirected to the event page in a browser.

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
- create an event and when you receive a notification
- you should see a button `View Details`.
- when you click on it it should redirect you to the browser showing details about the event.

#### What are the relevant github issues?
N/A

#### Screenshot(s)
